services/rds.py

```diff
@@ -1,95 +1,290 @@
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("AutoMinorVersionUpgrade", None) == True:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_default_admin_check():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("MasterUsername", None) not in ["admin", "postgres"]:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_deletion_protection_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("DeletionProtection", None) == True:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_encrypted_at_rest():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("StorageEncrypted", None) == True:
+            compliant_resources.append(cluster["DBClusterArn"])
+        else:
+            non_compliant_resources.append(cluster["DBClusterArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def rds_cluster_iam_authentication_enabled():
+    compliant_resources = []
+    non_compliant_resources = []
+    clusters = client.describe_db_clusters()["DBClusters"]
+
+    for cluster in clusters:
+        if cluster.get("IAMDatabaseAuthenticationEnabled", None) == True:
```

Comment:
DocDB에는 적용되지 않는 rule입니다.
```suggestion
        if cluster["Engine"] == "docdb" or cluster.get("IAMDatabaseAuthenticationEnabled"):
```